pkg/northbound/gnmi/set.go

```diff
@@ -351,52 +359,6 @@ func (s *Server) checkForReadOnly(target string, deviceType devicetype.Type, ver
 	return nil
 }
 
-func listenAndBuildResponse(mgr *manager.Manager, changeID networkchange.ID) ([]*gnmi.UpdateResult, error) {
-	networkChan := make(chan stream.Event)
-	ctx, errWatch := mgr.NetworkChangesStore.Watch(networkChan, networkchangestore.WithChangeID(changeID))
-	if errWatch != nil {
-		return nil, fmt.Errorf("can't complete set operation on target due to %s", errWatch)
-	}
-	defer ctx.Close()
-	updateResults := make([]*gnmi.UpdateResult, 0)
-	for changeEvent := range networkChan {
-		change := changeEvent.Object.(*networkchange.NetworkChange)
-		log.Infof("Received notification for change ID %s, phase %s, state %s", change.ID,
-			change.Status.Phase, change.Status.State)
-		if change.Status.Phase == changetypes.Phase_CHANGE {
-			switch changeStatus := change.Status.State; changeStatus {
-			case changetypes.State_COMPLETE:
```

Comment:
as per our discussion today we need to move this to changetypes.State_PENDING so we know atomix received the change. We can remove line 388 and passthrough with no action on any other events